google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/facet/flexible/FlexibleFacetEditor.java

```diff
@@ -302,7 +318,19 @@ public void actionPerformed(ActionEvent event) {
           }
         }
 
-        configFileGenerator.run();
+        switch (configType) {
+          case APP_YAML:
+            APP_ENGINE_PROJECT_SERVICE.generateAppYaml(FlexibleRuntime.JAVA, module,
+                destinationFolderPath);
+            break;
+          case DOCKERFILE:
+            APP_ENGINE_PROJECT_SERVICE.generateDockerfile(
```

Comment:
I think you need to merge in master so you get the logic that switches between jar and war dockerfiles instead of hardcoding it.